Sources/CorePayments/Networking/Enums/HTTPHeader.swift

```diff
@@ -7,4 +7,5 @@ public enum HTTPHeader: String {
     case authorization = "Authorization"
     case contentType = "Content-Type"
     case origin = "Origin"
+    case paypalClientContext = "paypal-client-context"
```

Comment:
```suggestion
    case payPalClientContext = "paypal-client-context"
```
nit: I'm honestly back and forth on which version is best.